### PR TITLE
Manpreet07 southborough

### PR DESCRIPTION
**Pull request by manpreet07**
This is the beginning of the Orlando file

Please provide feedback on

- if you have never been to Orlando, what's bring you here
- if you have been to Orlando, what thing you like most
- are you interested in staying here?
- @crichid what your child like to do here?
- @beardofedu whats you drink choices here?

### CODE REVIEW

File: manpreet07.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+##Southborough, MA
+
+##### *Southborough is a town in Worcester County, Massachusetts, United States. It incorporates the smaller villages of Cordaville, Fayville, and Southville*
```
Comment (diarmaidm) [inline manpreet07.md]: _Brilliant information_

**Comment by diarmaidm** [comment]
#### Nice formatting

File: manpreet07.md (hunk 1 above)
Comment (manpreet07) [inline manpreet07.md]: No Comments

**Comment by diarmaidm** [comment]
cool :+1:

**Pull request merged**